src/servers/Http1Server.cc

```diff
@@ -394,12 +392,14 @@ Http::One::Server::noteTakeServerConnectionControl(ServerConnectionContext serve
 ConnStateData *
 Http::NewServer(const MasterXaction::Pointer &xact)
 {
+    debugs(11, 2, "switch to HTTP on: " << xact->tcpClient);
     return new Http1::Server(xact, false);
 }
 
 ConnStateData *
 Https::NewServer(const MasterXaction::Pointer &xact)
 {
+    debugs(11, 2, "switch to HTTP on: " << xact->tcpClient);
```

Comment:
Moving outdated https://github.com/squid-cache/squid/pull/1105#discussion_r932224018 to new code:

> Alex: Please remove this line. This change is highly controversial (e.g., I do not think we are switching to HTTP here) and lies outside the declared PR scope.

> Amos: An HTTP server AsyncJob is being created and assigned ownership of the connection. This should not be controversial.

IMO, this code does not switch to HTTP and arguably does not (or should not) even create an HTTP server job. You may disagree, of course, but that disagreement is what makes this out-of-scope change controversial.

I support tracing protocol switches, but doing that right requires more work, and that work is outside _this_ PR scope. We want minimal, tightly-focused PRs, right? Removing this and the remaining "switch" tracing lines will move this PR closer to that ideal and avoid lengthy unnecessary discussions!